Library/Homebrew/dev-cmd/ruby.rb

```diff
@@ -1,13 +1,30 @@
 #:  * `ruby` [<ruby options>]:
 #:    Run a Ruby instance with Homebrew's libraries loaded.
-#:    For example:
-#       brew ruby -e "puts :gcc.f.deps"
-#       brew ruby script.rb
+#:
+#:    *Example:* `brew ruby -e "puts :gcc.f.deps"` or `brew ruby script.rb`
 
 module Homebrew
   module_function
 
+  def ruby_args
+    Homebrew::CLI::Parser.new do
+      usage_banner <<~EOS
+        `ruby` [<ruby options>]:
+
+        Run a Ruby instance with Homebrew's libraries loaded.
+
+        *Example:* `brew ruby -e "puts :gcc.f.deps"` or `brew ruby script.rb`
+      EOS
+      switch "-e",
+        description: "Execute the provided string argument as a script."
+      switch :verbose
```

Comment:
@EricFromCanada I think we'd have to make it `switch "-v"` instead of `switch :verbose` as it would automatically use `-v` but we want `-v` to print out Ruby version here.

Reply:
Seems to work as-is for me.
```
$ ruby -v
ruby 2.5.1p57 (2018-03-29 revision 63029) [x86_64-darwin15]
$ brew ruby -v
ruby 2.3.7p456 (2018-03-28 revision 63024) [universal.x86_64-darwin9.0]
```